include/dpp/webhook.h

```diff
@@ -118,6 +118,13 @@ class DPP_EXPORT webhook : public managed, public json_interface<webhook> {
 	 */
 	utility::iconhash avatar;
 
+	/**
+	 * @brief Avatar URL to use instead of the default if it is set.
+	 *
+	 * @note This may be empty.
```

Comment:
you might want to add something in here that says if it is set it supersedes the avatar variable. These two are now mutually exclusive.